pkg/graphite.go

```diff
@@ -74,6 +74,26 @@ func (m *Metrics) Init() error {
 	return nil
 }
 
+// RecordByLogTypes sends count of different types of logs
+func (m *Metrics) RecordByLogTypes(logType string) {
+	metricNameByHost := fmt.Sprintf("log_level.%s.by_host.%s.count.hourly", logType, quote(m.Hostname))
+	goMetrics.GetOrRegisterGauge(metricNameByHost, m.EveryHourRegister).Update(int64(1))
```

Comment:
If you want to calculate number of event, you need a counter (or meter), not a gauge:
https://godoc.org/github.com/rcrowley/go-metrics#Counter
https://godoc.org/github.com/rcrowley/go-metrics#Meter

In current case on every invocation it will just record "1" at current timestamp it's not what we want, we want to count number of error/warn records

Also hourly probably is too big, we want to monitor errors near to realtime, we need more frequent resolution, minutely I adice

The same applies to every other sended gauge

Also I'd advice to create unit test like metrics_test.go in gossip in order to lock that functionality